pycross/private/package_annotation.bzl

```diff
@@ -0,0 +1,24 @@
+"""Implementation of pycross_package_annotation"""
+
+def pycross_package_annotation(
+        always_build = False,
+        build_dependencies = [],
+        build_target_override = None,
```

Comment:
maybe this should be renamed to `build_target` to be symmetric to `lock_import.package` ?
if so, maybe that change should be reflected in `class PackageAnnotation`

Reply:
done